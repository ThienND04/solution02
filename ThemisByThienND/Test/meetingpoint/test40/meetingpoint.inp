7 13 6
5 4 8
4 7 17
3 5 1
3 4 9
3 7 8
3 2 10
5 3 1
1 3 12
2 6 17
5 1 11
6 2 20
7 6 1
3 4 9
5 5
1 4
8 8
3 6
4 4
7 3
