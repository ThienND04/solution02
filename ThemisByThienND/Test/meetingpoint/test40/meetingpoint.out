55
20
88
48
44
60
